src/data/locales/en.ts

```diff
@@ -1,28 +1,39 @@
 export default {
   "profile.trustButton": "Trust",
```

Comment:
The two lines below is what I added, the rest is just prettier auto-formatting this file